common/internal_model/src/main/resources/identifier-schemes.csv

```diff
@@ -14,3 +14,4 @@ preservica-guid,Perservica GUID
 calm-authority-code,CALM authority code
 isbn,International Standard Book Number
 issn,ISSN
+mets,METS

```

Comment:
I remember some talk about whether adding a new identifier type was the correct thing. Has @jtweed got any opinion on this?

Reply:
We discussed this with @jamesgorrie and I think we sort of reached the conclusion that just adding this here was OK?